### PR TITLE
Proposed improvements to install doc

### PR DESCRIPTION
**Pull request by abulte**
This stems from my attempt to install uData on a fresh install of Debian Jessie today.

### CODE REVIEW

File: docs/system-dependencies.md [hunk 1]
```diff
@@ -35,7 +36,7 @@ On any Debian-like system you can install the development tools and libraries wi
 $ apt-get install build-essential pkg-config python python-dev python-pip python-virtualenv\
     libjpeg-dev zlib1g-dev libtiff5-dev libfreetype6-dev \
     liblcms2-dev libopenjpeg-dev libwebp-dev libpng12-dev \
-    libxml2-dev  libxslt1-dev liblzma-dev libyaml-dev
+    libxml2-dev  libxslt1-dev liblzma-dev libyaml-dev libffi-dev
```
Comment (noirbizarre) [inline docs/system-dependencies.md]: Weird, I was sure I added it at the same time I added this one to the docker image: https://github.com/opendatateam/udata-dockers/blob/master/system/Dockerfile#L26

File: docs/system-dependencies.md [hunk 2]
```diff
@@ -58,14 +59,42 @@ Elasticsearch requires the [Analysis ICU][analysis-icu] plugin for your specific
...
+$ apt-get install redis-server
+```
+
+### MongoDB
+
+On Debian Jessie (cf [mongo-install-instructions][] for other versions), as root:
+
+```
+$ apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv EA312927
+$ echo "deb http://repo.mongodb.org/apt/debian jessie/mongodb-org/3.2 main" | sudo tee /etc/apt/sources.list.d/mongodb-org-3.2.list
```
[the remark this answers is not in the document]
Comment (abulte) [reply docs/system-dependencies.md]: 👍

File: docs/system-dependencies.md [hunk 3]
```diff
@@ -58,14 +59,42 @@ Elasticsearch requires the [Analysis ICU][analysis-icu] plugin for your specific
...
     deb https://packages.elastic.co/elasticsearch/2.x/debian stable main
 
 and install it like any other Debian package (You might need to enable Debian Backport repository)
 The `plugin` command is located in the `/usr/share/elasticsearch/bin` directory.
 
-For example, on Debian Jessie, you will have to perform (as root):
+You need a `java` binary in your path for ElasticSearch to operate properly, for example on Debian (as root):
+
+```shell
+$ apt-get install default-jre
```
[the remark this answers is not in the document]
Comment (abulte) [reply docs/system-dependencies.md]: I do not think it's a dependency (can't see it with `apt show` and did not install java on my box). I'll add the link though, always better.

File: docs/installation.md [hunk 4]
```diff
@@ -41,8 +41,9 @@ you log into its account:
 
 ```shell
 $ virtualenv --python=python2.7 $HOME
-$ source bin/activate  
+$ . bin/activate
```
Comment (oncletom) [inline docs/installation.md]: Out of curiosity, what does the `dot` approach changes to the `source` one?

Comment (noirbizarre) [reply docs/installation.md]: `.` is available in any `shell` including `sh`, `ash`... whereas `source` is only available in `bash`.

**Pull request merged**